### PR TITLE
Fix homepage links

### PR DESCRIPTION
**Pull request by elwoodxblues**

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
               <div class="home--content-wrapper">
                 <div class="home--content-inner">
                   <h1>{% trans "Promo &amp; Sale" %}</h1>
-                  <h2>{% trans "form the Northpole" %}</h2>
-                    <a class="btn btn-secondary home__button" href="/products/category/groceries-5/">{% trans "Shop now" %}</a>
+                  <h2>{% trans "from the Northpole" %}</h2>
```
Comment (patrys) [inline templates/home.html]: Maybe we can also change it to say "North Pole"?

**Comment by patrys** [comment]
Sorry for reviewing this in parts but I think it may also be a good opportunity to provide meaningful context to these translations.

**Comment by elwoodxblues** [comment]
@patrys I've added context to those translations, take a look if this is OK.

**Comment by elwoodxblues** [comment]
I'll merge this PR as I need it to release the new demo. In case of further changes I'll open separate PR.

**Pull request merged**